README.md

```diff
@@ -21,7 +21,49 @@ theme: minima
 And then execute:
 
     $ bundle
+
     
+## Contents At-A-Glance
```

Comment:
> As a new user, I don't think I was concerned with the files and directories until I wanted to customize something specific.

That's true.. but I don't feel it's right to move it 'after' the Customization section.. Start with the basics, know what's what.. then proceed to customize..